src/main/resources/com/sumologic/sumobot/http_frontend/script.js

```diff
@@ -18,23 +18,39 @@
  */
 window.onload = function() {
```

Comment:
```suggestion
window.addEventListener('load', ev => {/* ... */});
```
You don't risk overriding someone else's `load` handler that way.